letsdrill.rb

```diff
@@ -1,12 +1,18 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+	grades = { "A" => 90, "B" => 80, "C" => 70, "D" => 60 }
```

Comment:
Seems like a working solution. The grade is the key, and the code iterates through each value and finds a matching key. 

What if you try a different shape for the mapping; what about using [ranges](http://ruby-doc.org/core-2.2.0/Range.html) as keys, and the grades as values? i.e.

```ruby
grades = { (90..100) => "A", (80..89) => "B" }
```

What would the code look like to `#get_letter_grade` with the above structure?